src/appkit.cs

```diff
@@ -23683,4 +23687,20 @@ interface NSToolTipOwner
 		[Export ("view:stringForToolTip:point:userData:")]
 		string GetStringForToolTip (NSView view, nint tag, CGPoint point, IntPtr data);
 	}
+
+	[Protocol]
+	interface NSUserInterfaceValidations
+	{
+		[Abstract]
+		[Export ("validateUserInterfaceItem:")]
+		bool ValidateUserInterfaceItem (INSValidatedUserInterfaceItem item);
+	}
+
+	[Protocol (IsInformal=true)]
+	interface NSMenuValidation
+	{
+		[Abstract]
+		[Export ("validateUserInterfaceItem:")]
+		bool ValidateUserInterfaceItem (INSValidatedUserInterfaceItem item);
+	}
```

Comment:
do we need both ? again it's not an API I ever used but the bug report mention one of them is needed - and it turns out they are identical (but in name) to implement.

Is there a case where the informal must be used that the _normal_ protocol would not cover ?